README.md

```diff
@@ -1,26 +1,24 @@
 # Installation
 
-Python >= 3.12 required.
-
-Make a new virtualenv.
+Requires:
+- Python >= 3.12
+- [`uv>=0.4.27,<0.5.0`](https://github.com/astral-sh/uv) installed globally.
 
 ```shell
-$ pip install -U pip
-$ pip install pipenv
-$ pipenv sync --dev
+$ uv sync --dev
 $ playwright install
 ```
 
 Run tests:
 
 ```shell
-$ pytest
+$ uv pytest
 ```
 
 Run the service:
 
 ```shell
-$ uvicorn postcode_lookup.app:app --reload
+$ uv uvicorn postcode_lookup.app:app --reload
```

Comment:
Should these be `uv run pytest` and `uv run uvicorn...` ?

Reply:
https://github.com/DemocracyClub/ec-postcode-lookup-pages/pull/117/commits/f4fde1e5d882422015272e620e18d1dac4c05890